browser/ui/webui/brave_rewards_ui.cc

```diff
@@ -472,11 +472,7 @@ void RewardsDOMHandler::OnGrant(
 
 void RewardsDOMHandler::GetGrants(const base::ListValue* args) {
   if (rewards_service_) {
-    std::string lang;
-    std::string paymentId;
-    args->GetString(0, &lang);
-    args->GetString(1, &paymentId);
-    rewards_service_->FetchGrants(lang, paymentId);
+    rewards_service_->FetchGrants(std::string(), std::string());
```

Comment:
Is this change needed? This removes option so that JS could send lang and payment id

Reply:
There is no call anywhere from the Rewards UI that pass these params (this is how it is in the panel as well) 

If you would like to keep it how it was for possible future use I am fine with that.

Reply:
yup I know that we currently we don't use it, but if we already have it I would keep it

Reply:
@NejcZdovc reverted back

Reply:
awesome thx